dts/bindings/sensor/sensor-device.yaml

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: Sensor Device
+
+include: base.yaml
+
+properties:
+  friendly-name:
+    type: string
+    description: |
+      Human readable string describing the sensor. It can be used in a HID
+      sensor class descriptor to enumerate a sensor to a host operating system
```

Comment:
1. HID - I assume this is some spec or something that is implied here.  Can we expand on the reference of what HID means a little.
2. 'enumerate' doesnt feel correct, how about 'distinguish'?

Reply:
This was helpful for me https://docs.kernel.org/hid/index.html

Reply:
I don't know if makes sense to reference https://usb.org/sites/default/files/hut1_3_0.pdf specifically.  But I think some mention to it would be good (assuming that is where `friendly-name` is coming from

Reply:
Updated to:
```yaml
    description: |
      Human readable string describing the sensor. It can be used to
      distinguish multiple instances of the same model (e.g., lid accelerometer
      vs. base accelerometer in a laptop) to a host operating system.

      This property is defined in the Generic Sensor Property Usages of the HID 
      Usage Tables specification
      (https://usb.org/sites/default/files/hut1_3_0.pdf, section 22.5).
```